providers/component-protocol/cptype/sdk.go

```diff
@@ -37,19 +38,42 @@ type SDK struct {
 	Event       ComponentEvent
 	CompOpFuncs map[OperationKey]OperationFunc
 	Comp        *Component
+
+	// for parallel use, it's request level
+	StdStructuredPtr IStdStructuredPtr
+
+	Lock       sync.Mutex
+	OriginLock *sync.Mutex
```

Comment:
should not put Origin SDK's Lock into new SDK, i suggest you can define an OriginSDK aside with SDK

Reply:
adjusted